File: app/src/main/java/org/mifos/mobilebanking/models/accounts/Account.java
package org.mifos.mobilebanking.models.accounts;

import com.google.gson.annotations.SerializedName;

/**
 * This class contains variables which are used for implementing Comparator and are common to loan,
 * savings and share account.
 * Created by dilpreet on 14/6/17.
 */

public abstract class Account {
    @SerializedName("id")
    protected long id;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }
}

File: core/model/src/main/java/org/mifos/mobile/core/model/entity/accounts/Account.kt
/*
 * Copyright 2024 Mifos Initiative
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 *
 * See https://github.com/openMF/mobile-mobile/blob/master/LICENSE.md
 */
package org.mifos.mobile.core.model.entity.accounts

/**
 * This class contains variables which are used for implementing Comparator and are common to loan,
 * savings and share account.
 * Created by dilpreet on 14/6/17.
 */
abstract class Account {
    var id: Long = 0
}
